pokebattle/settings/test.py

```diff
@@ -19,6 +20,11 @@
 DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
 
+PASSWORD_HASHERS = [
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+]
+
 # Celery
 CELERY_TASK_ALWAYS_EAGER = True
 CELERY_TASK_EAGER_PROPAGATES = True
+BROKER_CAKEND = 'memory'
```

Comment:
should this be `BROKER_BACKEND`?

Reply:
I added that because of this [pull request](https://github.com/vintasoftware/django-react-boilerplate/pull/198) to help improving tests speed but turned out it's not recommended (I should've waited for it to be merged, my mistake)